docs/source/el7toel8/deprecation.md

```diff
@@ -10,9 +10,13 @@ after which the related functionality can be removed at any time.
 repository only. For the functionality deprecated in the leapp
 framework, see [List of deprecated functionality in leapp](../deprecation.html#list-of-deprecated-functionality-in-leapp)
 
-## current upstream development
+## current upstream development <span style="font-size:0.5em; font-weight:normal">(till ?? the next release + 6months)</span>
```

Comment:
What does the two question marks want to tell here?

Reply:
fixed